src/main/java/com/Studentlifr/GratitudeBoard/services/impl/AdminServicesImpl.java

```diff
@@ -0,0 +1,71 @@
+package com.Studentlifr.GratitudeBoard.services.impl;
+
+import com.Studentlifr.GratitudeBoard.dao.MessageDAO;
+import com.Studentlifr.GratitudeBoard.dao.UserDAO;
+import com.Studentlifr.GratitudeBoard.dao.repository.MessageRepository;
+import com.Studentlifr.GratitudeBoard.dao.repository.UserRepository;
+import com.Studentlifr.GratitudeBoard.dto.response.*;
+import com.Studentlifr.GratitudeBoard.enumerators.ROLES;
+import com.Studentlifr.GratitudeBoard.services.AdminServices;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@Service
+public class AdminServicesImpl implements AdminServices {
+    private final MessageRepository messageRepository;
+    private final UserRepository userRepository;
+    public AdminServicesImpl(MessageRepository messageRepository, UserRepository userRepository) {
+        this.messageRepository = messageRepository;
+        this.userRepository = userRepository;
+    }
+    public GenericResDTO<List<AdminMessagesResDTO>> getAllMessages() {
+        List<MessageDAO> messageDAOList = messageRepository.findAll();
+        List<AdminMessagesResDTO> AdminMessagesResDTOList = new ArrayList<>();
+
+        for (MessageDAO messageDAO : messageDAOList) {
+            AdminMessagesResDTO adminMessagesResDTO = new AdminMessagesResDTO();
```

Comment:
Consider using `ModelMapper` to simplify object mapping and improve code readability.
```suggestion
            AdminMessagesResDTO adminMessagesResDTO = modelMapper.map(messageDAO,AdminMessagesResDTO.class);
```